### PR TITLE
Improve buttons

### PR DESCRIPTION
**Pull request by ximex**
Done:
- Change tag type from `input` to `button`
- remove unused classes

TODO:
- Unify from l10n (save, edit, delete, ...)
- Add Icons
- Use colors

### CODE REVIEW

File: resources/views/pages/user-shifts.html [hunk 1]
```diff
@@ -52,7 +52,7 @@ <h1>%title%</h1>
             </div>
             <div class="row d-print-none">
                 <div class="col-md-12">
-                    <input class="btn btn-primary" type="submit" style="width:100%; margin: 1em 0" value="%filter%">
+                    <button class="btn btn-primary" type="submit" style="width:100%; margin: 1em 0" value="">%filter%</button>
```
Comment (frischler) [inline resources/views/pages/user-shifts.html:55]: You can probably remove the value="", too.

Comment (ximex) [reply resources/views/pages/user-shifts.html]: thx overlooked that. fixed

**Comment by ximex** [comment]
will do the other changes in another PR later

**Comment by ximex** [comment]
@MyIgel @weeman1337 ready for review. will do the other button changes another time

**Comment by MyIgel** [comment]
This has conflicts too which i can't fix which is anoying...

**Pull request closed without merging**